.travis.yml

```diff
@@ -5,8 +5,11 @@ sudo: false
 install:
       - make setup
       - make update
+      - nvm install 6.10
+      - nvm use 6.10
```

Comment:
Travis has node 0.10 installed by default. This causes errors because ESLint uses `const` declarations.